misc/h5m_explorer.py

```diff
@@ -0,0 +1,172 @@
+import sys
+import os
+
+import pandas as pd
+import numpy as np
+
+from pymoab import core
+from pymoab import types
+from pymoab.tag import Tag
+
+
+class h5mexplorer:
+
+    def __init__(self, filename):
+        """ Constructor """
+
+        self.mb = core.Core()
+        self.h5m_file = filename
+        self.mb.load_file(filename)
+
+        self.root_set = self.mb.get_root_set()
+        self.handles = self.mb.get_entities_by_handle(self.root_set)
+
+        self.materials = BuildMaterialTable()
+        self.volumes = BuildVolumeTable()
+
+    def GetCATEGORYHandles(self, category):
+        """
+        Get the list of handle with the CATEGORY TAG matching the user
+        spacification
+
+        Parameters
+        ----------
+        category : string
+            Value of the category tag to select
+
+        Returns
+        -------
+            MOAB Range of EntityHandles
+
+        """
+        category_tg_name = self.mb.tag_get_handle(
+            "CATEGORY", 32, types.MB_TYPE_OPAQUE)
+        return self.mb.get_entities_by_type_and_tag(self.root_set, types.MBENTITYSET,
+                                                    category_tg_name, np.array([category]))
+
+    def BuildMaterialTable(self):
+        """
+        Build the Material Table containing the list of material and one line
+        per assignment.
+
+        Returns
+        -------
+            Pandas DataFrame
```

Comment:
```suggestion
            mat_pdf: Pandas DataFrame
```

Since a variable is returned, it might be useful to have that variables name in the function docstring.